src/sonic-yang-models/tests/yang_model_tests/tests_config/fine_grained_ecmp.json

```diff
@@ -0,0 +1,288 @@
+        "sonic-fine-grained-ecmp:sonic-fine-grained-ecmp": {
+            "sonic-fine-grained-ecmp:FG_NHG": {
+                "FG_NHG_LIST": [
+                    {
+                        "name": "group1",
+                        "bucket_size": 10,
+                        "match_mode": "nexthop-based"
+                    },
+                    {
+                        "name": "group2",
+                        "bucket_size": 20,
+                        "match_mode": "route-based"
+                    }
+                ]
+            },
+            "sonic-fine-grained-ecmp:FG_NHG_PREFIX": {
+                "FG_NHG_PREFIX_LIST": [
+                    {
+                        "ip_prefix": "10.0.0.0/24",
+                        "FG_NHG": "group1"
+                    },
+                    {
+                        "ip_prefix": "10.1.0.0/24",
+                        "FG_NHG": "group2"
+                    }
+                ]
+            }
+        }
+    },
+    "FG_NHG_MEMBER_TEST": {
+        "sonic-port:sonic-port": {
+            "sonic-port:PORT": {
+                "PORT_LIST": [
+                    {
+                        "admin_status": "up",
+                        "alias": "eth0",
+                        "description": "Ethernet0",
+                        "lanes": "65",
+                        "mtu": "9000",
+                        "name": "Ethernet1",
+                        "tpid": "0x8100",
+                        "speed": "25000"
+                    }
+                ]
+            }
+        },
+        "sonic-fine-grained-ecmp:sonic-fine-grained-ecmp": {
+            "sonic-fine-grained-ecmp:FG_NHG": {
+                "FG_NHG_LIST": [
+                    {
+                        "name": "group1",
+                        "bucket_size": 10,
+                        "match_mode": "nexthop-based"
+                    },
+                    {
+                        "name": "group2",
+                        "bucket_size": 20,
+                        "match_mode": "route-based"
+                    }
+                ]
+            },
+            "sonic-fine-grained-ecmp:FG_NHG_MEMBER": {
+                "FG_NHG_MEMBER_LIST": [
+                    {
+                        "next_hop_ip": "10.0.0.1",
+                        "FG_NHG": "group1",
+                        "bank": 1,
+                        "link": "Ethernet1"
+                    },
+                    {
+                        "next_hop_ip": "10.0.0.2",
+                        "FG_NHG": "group1",
+                        "bank": 2,
+                        "link": "Ethernet1"
+                    },
+                    {
+                        "next_hop_ip": "10.1.0.1",
+                        "FG_NHG": "group2",
+                        "bank": 3,
+                        "link": "Ethernet1"
+                    }
+                ]
+            }
+        }
+    },
+    "FG_NHG_TEST_DUPLICATE_NAME": {
```

Comment:
Whats the difference between this one and FG_NHG_TEST_DUPLICATE_ENTRY, looks pretty similar to me as a test case

Reply:
removed